src/test/java/co/elastic/logstash/filters/elasticintegration/geoip/IpDatabaseProviderTest.java

```diff
@@ -131,7 +131,7 @@ static void withVendoredGeoIpDatabaseProvider(final ExceptionalConsumer<IpDataba
     }
 
     static void withGeoipProcessor(final IpDatabaseProvider geoIpDatabaseProvider, Map<String, Object> config, ExceptionalConsumer<Processor> geoIpProcessorConsumer) throws Exception {
-        Processor processor = new GeoIpProcessor.Factory("geoip", geoIpDatabaseProvider).create(Map.of(), null, null, config);
+        Processor processor = new GeoIpProcessor.Factory("geoip", geoIpDatabaseProvider).create(Map.of(), null, null, config, null);
```

Comment:
Was this previously failing to compile?

Reply:
Here is the answer: https://github.com/elastic/logstash-filter-elastic_integration/issues/305

Reply:
Can you please split this into a separate commit?

Reply:
Sorry missed addressing your request.
I have separated this change into [this PR](https://github.com/elastic/logstash-filter-elastic_integration/pull/306) to see and discuss to apply Java unit tests run. Let's move there to continue this topic.